### ISSUE TITLE
Backend Module shows "Sorry, the requested view was not found." in TYPO3 9.5

### ISSUE THREAD

**Issue by peerlight** [itx-informationssysteme/jobapplications#43]
Since update to version 1.0.2 the backend module only shows 

"Sorry, the requested view was not found. The technical reason is: No template was found. View could not be resolved for action "listApplications" in class "ITX\Jobapplications\Controller\BackendController"." 

in an installation with TYPO3 9.5.28

No problem in another installation with TYPO3 10.4

**Comment by itx-benjamin-jasper** [comment]
Works fine for me, did you include the static template?

**Comment by peerlight** [comment]
Yes, static template is loaded (among others) in root ts template which is on root page of website which is in a subfolder of TYPO3 root page (0)

checked this today:
- PHP-Version 7.2.24 
- TYPO3 9.5.30
- jobapplications 1.0.3

**Comment by itx-benjamin-jasper** [comment]
Maybe this is a PHP version issue, my 9.5 environment uses PHP 7.3. I have added the php version as a requirement to the composer.json for the next minor release. Closing this for now, please reopen if its still an issue with PHP 7.3.

**Comment by thomas-burger-de** [comment]
Changing the PHP version had no effect. BTW: In my opinion this would also affect all other frontend plugins / backend modules, since finding the template paths and files is a matter of the extbase framework.

In our setup of a TYPO3 installation (with multiple site roots) I found another explanation regarding this error.

Our installation contains several websites (site roots). And we don't use the extension jobapplications in the first site created in the page tree. Of course, in the website template of another website further down in the site tree where we use jobapplications, we included the Static TypoScript Template of the extension. And here the template paths for the backend module are specified (module.tx_jobapplications_web_jobapplicationsbackend.view.templateRootPaths.0 ... and so on ...).

After the error message "[...] View could not be resolved for action "listApplications" [...] appeared in the backend module, I finally included the Static TypoScript Template of the extension in the website template of the very first site root. Even if the extension jobapplications is not used in this particular website. Only then TYPO3 could know the paths for the templates of the backend module.
